lib/css_cleaner.rb

```diff
@@ -32,6 +32,11 @@ module CssCleaner
   DROP_SHADOW_VALUE_REGEX = Regexp.new("\\(\\s*(#{NUMBER_WITH_UNIT_REGEX}|#{COLOR_REGEX}\\s*)+\\s*\\)")
   DROP_SHADOW_FUNCTION_REGEX = Regexp.new("#{DROP_SHADOW_NAME_REGEX}#{DROP_SHADOW_VALUE_REGEX}")
 
+  # quotes can take up to 4 characters, wrapped in quotation marks and separated by spaces, including escaped tokens
```

Comment:
From what I can tell, literally [anything is allowed in the strings](https://developer.mozilla.org/en-US/docs/Web/CSS/quotes#text_as_quotes_and_empty_quotes) and there can be any number of pairs. With this code, the "They said" example I linked doesn't work.